forge/forge/op/tm.py

```diff
@@ -452,6 +452,29 @@ def Repeat(name: str, operandA: Tensor, factors: List[int]) -> Tensor:
     return op("repeat", name, operandA, attrs=factors).get_tensor()
 
 
+def RepeatInterleave(name: str, operandA: Tensor, repeats: int, dim: int) -> Tensor:
+    """
```

Comment:
Let's add more description about this op.

Reply:
Done